WebService/ComparisonEngineUI/ComparisonEngineUI/Views/DrinksPage.xaml

```diff
@@ -29,21 +29,6 @@
                     </ListView.ItemTemplate>
                 </ListView>
             </RefreshView>
-            <Label TextColor="Black" Text="Drink List:"
-                VerticalOptions="Center" 
-                HorizontalOptions="Center" />
-            <ListView ItemsSource="{Binding DrinkList}">
-                <ListView.ItemTemplate>
-                    <DataTemplate>
-                        <ViewCell>
-                            <StackLayout>
-                                <Button Text="{Binding DrinkName}" Clicked="Button_Clicked"/>
-                            </StackLayout>
-                        </ViewCell>
-                    </DataTemplate>
-                </ListView.ItemTemplate>
-            </ListView>
-            <Button Text="Edit" Command="{Binding EditDrinkCommand}"/>
```

Comment:
Why are we deleting this, isn't this the list of our drinks?

Reply:
This was left after bad merge conflict resolution.